exercises/hamming/tests/hamming.rs

```diff
@@ -1,14 +1,21 @@
 extern crate hamming;
 
+fn process_distance_case(strand_pair: [&str; 2], expected_distance: Option<usize>) {
```

Comment:
I'm curious why you wrote this as `[&str; 2]` instead of `(&str, &str)`. I tend to think of function arguments as tuple-like instead of array-like, so the latter would have seemed more natural to me.

I suspect that the two forms are equivalent in memory, and this does work; you certainly don't need to change it. I'm just curious about your design process.

Reply:
Guess I decided to go with the array, because the types of the variables are similar (perhaps a little C-like design).

If the strands where represented with different types, e.g. `LeftStrand` / `RightStrand`, then I would surely used the tuple.